script/lazytox.py

```diff
@@ -117,7 +117,9 @@ async def pylint(files):
 
 async def flake8(files):
     """Exec flake8."""
-    _, log = await async_exec("pre-commit", "run", "flake8", "--files", *files)
+    _, log = await async_exec(
+        "pre-commit", "run", "flake8", "--max-complexity", "25", "--files", *files
```

Comment:
This should not be needed, as it is specified in the setup.cfg

Reply:
Thanks.  should be sorted now.